### PR TITLE
Message attachment extended to match full Slack's format.

### PR DESCRIPTION
**Pull request by Shimmi**
As the Slack's attachment format is much bigger than just text, pretext, color and fields I've extended it to match all features. See https://api.slack.com/docs/attachments for docs.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 composer.lock
 composer.phar
 vendor
+.idea
```
Comment (DZunke) [inline .gitignore]: please do not put developer environment specific stuff in the project .gitignore. to get this ignored you should use your global .gitignore. you can set it to your git client by using:

```
git config --global core.excludesfile ~/.gitignore_global
```

**Comment by DZunke** [comment]
thanks for your work. i will check the code later this day and give response. would be nice if you change the .gitignore before

**Comment by Shimmi** [comment]
@DZunke My bad! Sorry. I forgot it there... gitignore reverted. Thanks for such a quick look at it!

**Comment by DZunke** [comment]
Merged and Released Patch v1.2.2

**Comment by Shimmi** [comment]
Awesome, thanks!

**Pull request merged**